spyder/utils/site/sitecustomize.py

```diff
@@ -280,6 +280,14 @@ def __init__(self, *args, **kwargs):
         TestProgram.__init__(self, *args, **kwargs)
 unittest.main = IPyTesProgram
 
+# Filter warnings that appear for ipykernel wheninteracting with
+# the Variable explorer (i.e trying to see a variable)
+# Fixes Issue 5591
+warnings.filterwarnings(action='ignore', category=DeprecationWarning,
+                        module='ipykernel.datapub',
+                        message=".*ipykernel.datapub is deprecated. "

```

Comment:
I think a single line is enough here. So it should be

    message=".*ipykernel.datapub is deprecated.*"